diff --git a/scripts/updatedb.js b/scripts/updatedb.js
index a8738225a3b58164758766a83223e1995538bde6..c5beeecbcaf41c0c4b9e6cb0124c29a1dc1423f0 100644
--- a/scripts/updatedb.js
+++ b/scripts/updatedb.js
@@ -384,8 +384,8 @@ async function processCountryData(src, dest) {
 				bsz = 10;
 
 				rngip = new Address4(fields[0]);
-				sip = parseInt(rngip.startAddress().bigInteger(),10);
-				eip = parseInt(rngip.endAddress().bigInteger(),10);
+				sip = parseInt(rngip.startAddress().bigInt(),10);
+				eip = parseInt(rngip.endAddress().bigInt(),10);
 	
 				b = Buffer.alloc(bsz);
 				b.fill(0);
@@ -494,8 +494,8 @@ async function processCityData(src, dest) {
 			bsz = 24;
 
 			rngip = new Address4(fields[0]);
-			sip = parseInt(rngip.startAddress().bigInteger(),10);
-			eip = parseInt(rngip.endAddress().bigInteger(),10);
+			sip = parseInt(rngip.startAddress().bigInt(),10);
+			eip = parseInt(rngip.endAddress().bigInt(),10);
 			locId = parseInt(fields[1], 10);
 			locId = cityLookup[locId];
 			b = Buffer.alloc(bsz);
